recipes/adpred/meta.yaml

```diff
@@ -16,17 +16,16 @@ build:
 
 requirements:
   host:
-    - python >=3.6
-    - setuptools==46
+    - python >=3.9
+    - setuptools >=51.0.0
   run:
-    - python >=3.6
-    - keras==2.2.4
-    - scikit-learn==0.21.3
-    - numpy==1.17.2
-    - plotly==4.1.1
-    - tensorflow==1.14.0
-    - requests==2.23.0
-    - pandas
+    - keras ==2.11.0
+    - scikit-learn ==1.1.3
+    - numpy ==1.23.5
+    - plotly ==5.11.0
+    - tensorflow ==2.11.0
+    - requests ==2.28.1
+    - pandas ==1.5.2
```

Comment:
You really need these exact versions and absolutely nothing else? That is, keras 2.11.1 would break things, as would pandas 1.5.1 or 1.5.3? I strongly suggest instead setting reasonable minimal versions (e.g., `keras >=2`).